home-manager/zsh.nix

```diff
@@ -231,6 +231,8 @@ in
 
       source "${pkgs.fzf-git-sh}/share/fzf-git-sh/fzf-git.sh"
       source "${pkgs.podman}/share/zsh/site-functions/_podman"
+      source "${pkgs.kubectl}/share/zsh/site-functions/_kubectl"
+      source "${pkgs.kind}/share/zsh/site-functions/_kind"
```

Comment:
These sourcing in nix layer having problem as https://github.com/kachick/dotfiles/pull/917 in darwin